addons/mail/static/src/web/activity/activity_menu.js

```diff
@@ -56,7 +57,7 @@ export class ActivityMenu extends Component {
         } else {
             let domain = [["activity_ids.user_id", "=", this.userId]];
             if (group.domain) {
-                domain = domain.concat(group.domain);
+                domain = Domain.and([domain, group.domain]).toList({});
```

Comment:
In `.toList({})`, `{}` is useless. Even if the docstring says `context` is mandatory, inner-code defaults to `{}` when context is not provided.

I would remove `{}` so this LOC with many brackets is more readable :)